README.md

```diff
@@ -18,7 +17,7 @@ For detailed information about payment setup (currently unsupported), check the
 
 ## Installation
 
-Proxeus is primarily a Web application, and is currently not intended for use on a desktop computer. The [Proxeus Association](https://proxeus.org) maintains a demo instances you can use to test the product, and can recommend a service provider to help you or your business get set up. 
+Proxeus is primarily a Web application, intended for access with a web browser. The [Proxeus Association](https://proxeus.org) maintains a demo instances you can use to test the product, and can recommend a service provider to help you or your business get set up. There is also a prototype [desktop application](https://github.com/ProxeusApp/storage-app/blob/master/docs/overview.md).
```

Comment:
@loleg not sure that we should rely on https://github.com/ProxeusApp/storage-app/ - bcs this repo is not used

Reply:
Yes, it is currently inactive and I have therefore archived the repo. As soon as we have a better example of an externally run client (desktop or web app) accessing Proxeus API, we can change this link.

Thanks for the feedback!